pkg/stack/stack_processor.go

```diff
@@ -59,15 +59,24 @@ func ProcessYAMLConfigFiles(
 			sort.Strings(uniqueImports)
 
 			componentStackMap := map[string]map[string][]string{}
-			if processStackDeps {
-				componentStackMap, err = CreateComponentStackMap(stackBasePath, p)
-				if err != nil {
-					errorResult = err
-					return
-				}
-			}
-
-			finalConfig, err := ProcessConfig(stackBasePath, p, config, processStackDeps, processComponentDeps, "", componentStackMap, importsConfig)
+			// TODO: this feature is not used anywhere, it has old code and it has issues with some YAML stack configs
+			// TODO: review it to use the new `atmos.yaml CLI config
+			//if processStackDeps {
+			//	componentStackMap, err = CreateComponentStackMap(stackBasePath, p)
+			//	if err != nil {
+			//		errorResult = err
+			//		return
+			//	}
+			//}
+
+			finalConfig, err := ProcessConfig(stackBasePath,
+				p,
+				config,
+				processStackDeps,
+				processComponentDeps,
+				"",
```

Comment:
Why is this empty ?

Reply:
it was like that :)
no component type filter, meaning to process all component types, terraform and helmfile